src/main/java/appeng/me/cluster/implementations/CraftingCPUCluster.java

```diff
@@ -1572,6 +1573,18 @@ public void togglePlayerFollowStatus(final String name) {
         } else {
             this.playersFollowingCurrentCraft.add(name);
         }
+
+        final Iterator<Entry<IMEMonitorHandlerReceiver<IAEItemStack>, Object>> i = this.getListeners();
+        // protect integrity
+        if (i.hasNext()) {
```

Comment:
What is the purpose of this if-statement, is this not covered by the while statement within?

Reply:
done